flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -553,11 +557,18 @@ private static void setTimeoutAfter(NotificationDetails notificationDetails, Not
 
     private static Class getMainActivityClass(Context context) {
         String packageName = context.getPackageName();
-        Intent launchIntent = context.getPackageManager().getLaunchIntentForPackage(packageName);
-        String className = launchIntent.getComponent().getClassName();
+        PackageManager packageManager = context.getPackageManager();
```

Comment:
Thanks for the PR. I just realised that given `getMainActivityClass` was only really to specify the intent relating to the launch activity, I suspect this entire method could be removed. That is, the intent from calling `packageManager.getLaunchIntentForPackage(packageName)` could be passed to the notification itself (i.e. [here](https://github.com/MaikuB/flutter_local_notifications/blob/88ff541da3bfe5af877724a472967467b731f6d6/flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java#L162). I imagine that should still solve the issue you had with activity aliases. If so, could you make the change I described in this PR?